src/main/flight/alt_hold.c

```diff
@@ -161,8 +161,8 @@ void altHoldProcessTransitions(altHoldState_s* altHoldState) {
     }
 
     uint32_t timeSinceExit = currTime - altHoldState->exitTime;
-    if (timeSinceExit < ALTHOLD_MAX_EXIT_PERIOD) {
```

Comment:
ALTHOLD_MAX_EXIT_PERIOD is correct. I just hate the logic

Reply:
Reverted